### PR TITLE
fix: typos/minor rewrites for events & add: new med cat moon events (misc)

### PR DESCRIPTION
**Pull request by capillarys**
caught a few typos so i changed em, and also changed duplicate events/added a handful more misc events!

### CODE REVIEW

File: resources/dicts/events/nutrition/deputy.json [hunk 1]
```diff
@@ -2,11 +2,11 @@
 	{
 		"camp": "any",
 		"tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "malnourished"],
-		"death_text": "m_c feels the crumbing of they stomach, but they try to stay strong for the clan and be a good example.",
+		"death_text": "m_c feels their stomach crumble, but they try to stay strong for the Clan and be a good example.",
```
Comment (Ryos00) [inline resources/dicts/events/nutrition/deputy.json]: "rumble" instead of "crumble"?

File: resources/dicts/events/nutrition/general.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	{
         "camp": "any",
         "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "death"],
-        "death_text": "m_c can feel they have no energy, not to stand up and not to remain longer here. m_c hunt now with their ancestors.",
+        "death_text": "m_c feels that they have no energy, unable stand up and remain longer here. m_c hunts now with their ancestors.",
```
Comment (Ryos00) [inline resources/dicts/events/nutrition/general.json]: "unable to stand up and unable to remain here any longer."

File: resources/dicts/events/misc_events/general/medicine.json [hunk 3]
```diff
@@ -17,4 +17,23 @@
...
+        "event_text": "m_c visits o_c to converse with their medicine cats about an omen StarClan sent.",
+        "cat_trait": ["strong connection to starclan", "omen sight", "clairvoyant", "dream walker"],
+        "cat_skill": ["compassionate", "faithful", "altruistic"],
+        "other_cat_trait": null,
+        "other_cat_skill": null
+    },
+{
+        "camp": "any",
+        "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "other_clan"],
+        "event_text": "m_c visits o_c to congratulate them on their new medicine cat."],
```
Comment (Ryos00) [inline resources/dicts/events/misc_events/general/medicine.json]: there's some weird stuff going on with your brackets here. this one in line 33 is unneeded, and some of the curly braces should be [] instead

File: resources/dicts/events/nutrition/deputy.json (hunk 1 above)
Comment (capillarys) [reply resources/dicts/events/nutrition/deputy.json]: ah yeah that's def better, i just found crumbing odd :P i'll change all the crumbing/crumbling to rumbling!

File: resources/dicts/events/misc_events/general/medicine.json (hunk 3 above)
Comment (capillarys) [reply resources/dicts/events/misc_events/general/medicine.json]: @Ryos00 i've changed some of the brackets and i hope i've gotten it right? i put the bracket on the wrong line for line 33, was meant to go into 34. which curly brackets do i need to change?

File: resources/dicts/events/nutrition/general.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
 	{
         "camp": "any",
         "tags": ["Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare", "death"],
-        "death_text": "m_c feels that they have no energy, unable stand up and remain longer here. m_c hunts now with their ancestors.",
+        "death_text": "m_c feels that they have no energy, unable to stand up and unable to remaine here any longer. m_c hunts now with their ancestors.",
```
Comment (Ryos00) [inline resources/dicts/events/nutrition/general.json]: remaine ;u;

Comment (capillarys) [reply resources/dicts/events/nutrition/general.json]: NAUR NOT REMAINE </3

**Review by Ryos00 — APPROVED** [review]

**Pull request merged**